phpcs.xml.dist

```diff
@@ -2,12 +2,19 @@
 <ruleset name="Pods">
 	<description>Pods-customized WordPress Coding Standards</description>
 
-	<exclude-pattern>.idea/*</exclude-pattern>
-	<exclude-pattern>.saas-cache/*</exclude-pattern>
-	<exclude-pattern>node_modules/*</exclude-pattern>
-	<exclude-pattern>tests/*</exclude-pattern>
-	<exclude-pattern>vendor/*</exclude-pattern>
-	<exclude-pattern>components/Markdown.php</exclude-pattern>
+	<file>.</file>
+	<exclude-pattern>*/.idea/*</exclude-pattern>
+	<exclude-pattern>*/.sass-cache/*</exclude-pattern>
+	<exclude-pattern>*/node_modules/*</exclude-pattern>
+	<exclude-pattern>*/tests/*</exclude-pattern>
```

Comment:
Shouldn't the files in the `/tests/` directory follow the same code standards as well?

Yes, these aren't provided to the plugin users, but they are (or should be) as much a part of the project as the source files.

Reply:
Yeah we can probably remove the tests exclusion